src/components/configurable-items/configurable-items.component.js

```diff
@@ -20,8 +20,8 @@ let deprecatedWarnTriggered = false;
 class ConfigurableItems extends React.Component {
   static contextType = LocaleContext;
 
-  constructor(props) {
-    super(props);
+  constructor(props, context) {
+    super(props, context);
```

Comment:
This should be in separate commit. Does this add any functionality or is it just code style?

Reply:
just code style, we can ignore it for now